### PR TITLE
Fix broken tests which relied on ex-team member

### PR DESCRIPTION
**Pull request by theKHutDeveloper**
Replace signed_in user with MOIC_POM, instead of relying on team members so that the majority of tests continue to work when team member leaves

### CODE REVIEW

File: spec/controllers/allocations_controller_spec.rb [hunk 1]
```diff
@@ -167,7 +167,26 @@
     end
 
     context 'without DeliusDataJob' do
-      render_views
```
Comment (starswan) [inline spec/controllers/allocations_controller_spec.rb:170]: think this was removed by accident?

Comment (starswan) [reply spec/controllers/allocations_controller_spec.rb:170]: don't think this line should have been deleted

Comment (theKHutDeveloper) [reply spec/controllers/allocations_controller_spec.rb:170]: I've put it back

Comment (ollietreend) [reply spec/controllers/allocations_controller_spec.rb]: I can see that this allocation fixture `before` block has been moved one level up into the parent `context 'without DeliusDataJob'` block, where it used to be inside the child `context 'with an allocation'` block.

It looks like we create a different allocation fixture in the other child context block `context 'with a different allocation'`, so I don't think it uses the first allocation fixture.

I'm wondering if it makes more sense to keep it in the `context 'with an allocation'` block. Or is there are reason for it being moved up into the parent?

File: spec/services/prison_offender_manager_service_spec.rb [hunk 2]
```diff
@@ -38,7 +38,7 @@
          vcr: { cassette_name: :pom_service_get_poms_list } do
         expect(subject).to be_kind_of(Enumerable)
         # 1 POM in T3 (Toby Retallick) is marked inactive, so expect one less active one
```
Comment (starswan) [inline spec/services/prison_offender_manager_service_spec.rb:40]: silly question, but why has this test changed? Did you remove Toby R as well?

**Comment by ollietreend** [comment]
@theKHutDeveloper it looks like you and Ross are hardcoded in the `PrisonOffenderManagerService` spec in the `#get_pom_name` and `#get_user_name` tests:
https://github.com/ministryofjustice/offender-management-allocation-manager/blob/8097da0478501fecfe6c72ea4fd590f4c8a4739d/spec/services/prison_offender_manager_service_spec.rb#L12-L27

**Pull request merged**